specs-go/v1/artifact.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
2022 :)

Reply:
:) Fixed. I was not sure if it was something to do with the license header.